source/configuration_manual/push_notification.rst

```diff
@@ -123,34 +133,35 @@ used by any push endpoint that implements this API, not just OX App Suite.
+|                                             |          |                |                                                                                                                                      |
+|                                             |          |                | For HTTPS endpoints, system CAs are trusted by default, but internal CAs might need further configuration.                           |
+|                                             |          |                |                                                                                                                                      |
+|                                             |          |                | For further details on configuring the App Suite endpoint, see:                                                                      |
+|                                             |          |                | https://documentation.open-xchange.com/latest/middleware/mail/dovecot/dovecot_push.html#configuration-of-dovecot-http-notify-plug-in |
++---------------------------------------------+----------+----------------+--------------------------------------------------------------------------------------------------------------------------------------+
+| ``push_notification_ox_cache_ttl``          | NO       | :ref:`time`    | Cache lifetime for the METADATA entry for a user. (DEFAULT: ``60 seconds``)                                                          |
++---------------------------------------------+----------+----------------+--------------------------------------------------------------------------------------------------------------------------------------+
+| ``push_notification_ox_user_from_metadata`` | NO       | :ref:`boolean` | Use the user stored in the METADATA entry instead of the user sent by OX endpoint.                                                   |
+|                                             |          |                | (DEFAULT: user returned by endpoint response is used, i.e. ``no``)                                                                   |
++---------------------------------------------+----------+----------------+--------------------------------------------------------------------------------------------------------------------------------------+
 
 Example configuration:
 
 .. code-block:: none
 
-  plugin {
-    push_notification_driver = ox:url=http://login:pass@node1.domain.tld:8009/preliminary/http-notify/v1/notify user_from_metadata timeout_msecs=10000
+  push_notification ox {
+    driver = ox
+    push_notification_ox_url = http://login:pass@node1.domain.tld:8009/preliminary/http-notify/v1/notify
+    push_notification_ox_user_from_metadata = yes
+    push_notification_ox_cache_ttl = 10secs
```

Comment:
These could be also simply `ox_url`, `ox_user_from_metadata`. Not quite sure whether that's nicer or not. Maybe more confusing..

Reply:
I opt for the long and explicit version. This makes it much more clear, without raising any further questions.

I _could_ add a note regarding "The `push_notification_` prefix is optional but suggested. This document will use the verbose version."